tests/Doctrine/Tests/Common/Proxy/LazyLoadableObjectWithTypehints.php

```diff
@@ -90,5 +93,10 @@ public function getIdentifierFieldReturnClassOneLetter(): A
         return $this->identifierFieldReturnClassOneLetter;
     }
 
+    public function getIdentifierFieldReturnClassOneLetterNullable(): ?A
```

Comment:
this must be done in a separate class, in a test being skipped on PHP < 7.1. Otherwise, it breaks the testsuite for PHP 7.0